resources/settings.xml

```diff
@@ -5,6 +5,13 @@
 		<setting id="reverseorder" label="40002" type="labelenum" values="Ascending|Descending" default="Descending"/>
 		<setting id="searchhistory" label="40003" type="labelenum" values="Yes|No" default="Yes"/>
 
+		<setting type="lsep" label="Login"/>
```

Comment:
So looks like we need to remove wonderful subs from the playlist flavor (as everone is logged in by default) and handle different flavor only if set..

Reply:
Also note that some changes leaked from this commit into previous one, like strings and forbiden/use token

Reply:
Seems like you can browse some lists without being logged in, so not sure, if we want to remove the watchlist flavor. Probably a good idea.

Reply:
You need it to play? What gives browse without play?

Reply:
Fair point. I usually just log in when I have to, but I'll try to put it on load.